### PR TITLE
New package: arc-icon-theme-20160605

### PR DESCRIPTION
**Pull request by vjon**

### CODE REVIEW

File: srcpkgs/arc-icon-theme/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'arc-icon-theme'
+pkgname=arc-icon-theme
+version=20160605
+revision=1
+noarch=yes
+build_style=gnu-configure
+configure_script="./autogen.sh"
+configure_args="--prefix=/usr"
```
Comment (Vaelatern) [inline srcpkgs/arc-icon-theme/template]: Already set by `build_style`

File: srcpkgs/arc-icon-theme/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+hostmakedepends="autoconf automake"
```
Comment (Vaelatern) [inline srcpkgs/arc-icon-theme/template]: https://gist.github.com/945460aa90503e1fb7a224d42754950c
`hostmakedepends="automake"`

File: srcpkgs/arc-icon-theme/template (hunk 1 above)
Comment (vjon) [inline srcpkgs/arc-icon-theme/template]: I've tried omitting the "configure_script" and "configure_args" lines, but the build fails when xbps-src attempts to run "./configure" in the directory. This was what led me to believe that these two lines were necessary.

Comment (Vaelatern) [inline srcpkgs/arc-icon-theme/template]: I was not arguing the setting of `configure_script`.

```
#
# This helper is for templates using GNU configure scripts.
#
do_configure() {
	: ${configure_script:=./configure}

	${configure_script} ${configure_args}
}

do_build() {
	: ${make_cmd:=make}

	${make_cmd} ${makejobs} ${make_build_args} ${make_build_target}
}

do_install() {
	: ${make_cmd:=make}
	: ${make_install_target:=install}

	${make_cmd} DESTDIR=${DESTDIR} ${make_install_args} ${make_install_target}
}
```

The above will work just fine if you just override the configure script.

**Pull request closed without merging**